### ISSUE TITLE
False positive behavior

### ISSUE THREAD

**Issue by sirian** [ljharb/is-callable#30]
```javascript
class A{}
isCallable(A) // false, ok

isCallable(new Proxy(A, {})) // true, should be false
isCallable(A.bind()) // true, should be false
isCallable(class{}) // true, should be false
isCallable(class/**/A{}) // true, should be false
```

Also you inproperly strip comments inside isES6ClassFunction.
In below example `var spaceStripped` === `class/*A{}`
```javascript
isCallable(class/*//*/
A{}) //true, should be false. 
```

```javascript
a = cl/*/**/=>/**/ass - 1;
isCallable(a); //false, should be true
```

**Comment by ljharb** [comment]
I believe I've corrected all your test cases in 33c090d19728b86327a483799fb5986cabdd1652, except for `bind` (which is supposed to hide information about the thing being bound), and the Proxy, which is captured by #25.

**Comment by sirian** [comment]
Still wrong
```
isCallable(class/*/*/A{})
isCallable(class/*//*//*//**/A{})
```

I've submitter #31
